keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/HalfCauchyVertex.java

```diff
@@ -44,6 +48,19 @@ public double logProb(DoubleTensor value) {
         return Double.NEGATIVE_INFINITY;
     }
 
+    @Override
+    public Map<Vertex, DoubleTensor> dLogProb(DoubleTensor value, Set<? extends Vertex> withRespectTo) {
```

Comment:
Was this just wrong before?

Reply:
Yeah - we have masks in place for the `logProb` but not `dLogProb`, we just missed it